Composer/packages/client/src/components/Header.tsx

```diff
@@ -291,34 +294,37 @@ export const Header = () => {
           </div>
         )}
         {isShow && (
-          <IconButton
-            ariaDescription={formatMessage('Test your bot')}
-            disabled={!webchatEssentials?.botUrl}
-            iconProps={{
-              iconName: 'OfficeChat',
-            }}
-            styles={buttonStyles}
-            title={formatMessage('Test your bot')}
-            onClick={() => {
-              const currentWebChatVisibility = !isWebChatPanelVisible;
-              setWebChatPanelVisibility(currentWebChatVisibility);
-              if (currentWebChatVisibility) {
-                TelemetryClient.track('WebChatPaneOpened');
-              } else {
-                TelemetryClient.track('WebChatPaneClosed');
-              }
-            }}
-          />
+          <TooltipHost content={labels().test} directionalHint={DirectionalHint.bottomCenter}>
+            <IconButton
+              ariaDescription={labels().test}
```

Comment:
You should call the `labels` function and unpack it instead of calling `labels` multiple times. 

```js
const {
  logo: logoLabel,
  test: testLabel,
  rocket: rocketLabel,
  update: updateLabel,
 } = labels();
```

```suggestion
              ariaDescription={testLabel}
```

Reply:
Good idea. I'll fix that.